pmm-app/tests/QAN/pages/qanPage.js

```diff
@@ -0,0 +1,136 @@
+const { I } = inject();
+const assert = require('assert');
+
+module.exports = {
+  url: 'graph/d/pmm-qan/pmm-query-analytics?from=now-5m&to=now',
+  filterGroups: [
+    'Environment',
+    'Cluster',
+    'Replication Set',
+    'Database',
+    'Node Name',
+    'Service Name',
+    'User Name',
+    'Node Type',
+    'Service Type',
+  ],
+  fields: {
+    table: '//table//tr[2]',
+    nextPageNavigation: '//ul[@role=\'navigation\']//li[last()]',
+    iframe: '//div[@class=\'panel-content\']//iframe',
+    newQANPanelContent: '.panel-content',
+    disabledResetAll: '//button[@data-qa="qan-filters-reset-all" and @disabled ]',
+    newQANSpinnerLocator: '//i[@data-qa=\'loading-spinner\']',
+    showSelected: '//button[@data-qa=\'qan-filters-show-selected\']',
+    countOfItems: '//span[@data-qa=\'qan-total-items\']',
+    filterBy: '//input[@data-qa=\'filters-search-field\']',
+    filterCheckboxes: '.checkbox-container__checkmark',
+    newQANAddColumn: '//span[contains(text(), \'Add column\')]',
```

Comment:
I suggest we use double quotes in such cases. This'd read better than escaping.
```suggestion
    newQANAddColumn: '//span[contains(text(), "Add column")]',
```

Reply:
Besides, it seems to be a general practice out there.

Reply:
will fix. thanks